tests/unit/src/main/scala/scala/meta/internal/metals/debug/Debugger.scala

```diff
@@ -65,7 +65,6 @@ final class Debugger(server: RemoteServer)(implicit ec: ExecutionContext) {
   def disconnect: Future[Unit] = {
     val args = new DisconnectArguments
     args.setRestart(false)
-    args.setTerminateDebuggee(false)
```

Comment:
This doesn't need to be specified, the server should take care of it appropriately. 

https://microsoft.github.io/debug-adapter-protocol/specification#Requests_Disconnect